plugin-maven/src/main/java/com/diffplug/spotless/maven/FormatterFactory.java

```diff
@@ -32,7 +32,7 @@
 import com.diffplug.spotless.Formatter;
 import com.diffplug.spotless.FormatterStep;
 import com.diffplug.spotless.LineEnding;
-import com.diffplug.spotless.maven.generic.LicenseHeader;
+import com.diffplug.spotless.maven.generic.*;
```

Comment:
I admit that I was surprised that this change from a full import to a wildcard import was apparently accepted by Travis CI.

@nedtwigg Do you think it would be a good idea if I open an issue to cover catching wildcard imports in the future?

Reply:
The eclipse formatter that we use doesn't have any rules against wildcard imports.  It might be worth a bullet-point in this issue, maybe a config option on importSorter if it ever gets updated: https://github.com/diffplug/spotless/issues/167

Reply:
@nedtwigg I'm not sure that I understood your last message correctly.

Did you mean that you'd like me to write a new message on #167 - a reminder of sorts - to add a convert-wildcard-imports-to-full-imports option to the new `importOrder()` formatting option that would be implemented? Or did you mean something else?

Reply:
Yup!  You can make a new issue too if you'd like, but it seems like implementing the wildcard imports issue is so closely tied to the ImportSorter that it makes sense to bundle them.

Reply:
Cool! I'll go act on that now, then. :)